docs/layers/tile-layer.md

```diff
@@ -173,19 +174,37 @@ How the tile layer refines the visibility of tiles. One of the following:
 
 - Default: `'best-available'`
 
+##### `throttleRequests` (Boolean, optional)
+
+Whether to throttle `getTileData` requests.
+
+If `true`, a maximum of `maxRequests` instances of `getTileData` will be called concurrently. Additionally, for requests that are throttled, if the tile is no longer visible in the viewport when a request slot opens up, that request will be cancelled. When requests are not throttled, `getTileData` will be called for all visible tiles, regardless of how long that tile is or was visible.
+
+- Default: `true`
+
+##### `maxRequests` (Number, optional)
```

Comment:
Can we merge these two props? I.e. if `maxRequest > 0` then enable `throttleRequests`.

Reply:
I figured that would come up during API review; I already set `throttleRequests` to `false` on the request scheduler when `maxRequest <=0`

https://github.com/visgl/deck.gl/blob/d96e91f56f2c120d59c1c5ee2832848bed9be139/modules/geo-layers/src/tile-layer/tileset-2d.js#L61

Reply:
I removed `throttleRequests` from the docs, pending #4727.